### PR TITLE
SPCTR-610 Improvement: Remove word "device" from the tooltip in responsive icons.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/b1c2d232-e4f3-418a-b107-2207d0ac04fb?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- https://share.getcloudapp.com/NQuNQkLB

### Types of changes

- Improvement

### How has this been tested?

- Click on the Responsive Icon in the editor and check the tooltip.
- Check other component like Responsive slider, Spacing and wherever we added responsive support.
- There should be device name on tooltip, no static string.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by RajkiranBagal** [comment]
No Prefix for PR title

**Comment by RajkiranBagal** [comment]
PR title not according to the task.

File: src/components/responsive-toggle/index.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ const devices = [
 						<Tooltip
 							text={ sprintf(
 								/* translators: abbreviation for units */
-								__( '%s device', 'ultimate-addons-for-gutenberg' ),
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/components/responsive-toggle/index.js:152]: Added translation here - https://share.getcloudapp.com/L1uR1orz

File: src/components/responsive-icons/device-icons.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ if ( ! deviceType ) {
 				<Tooltip
 					text={ sprintf(
 						/* translators: abbreviation for units */
-						__( '%s device', 'ultimate-addons-for-gutenberg' ),
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/components/responsive-icons/device-icons.js:99]: Added translation here - https://share.getcloudapp.com/L1uR1orz

**Comment by AbhijitNage123** [comment]
> No Prefix for PR title

Updated PR title

**Comment by AbhijitNage123** [comment]
> PR title not according to the task.

Updated title as per task

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**